src/Components/Posts/FeedSection.js

```diff
@@ -0,0 +1,104 @@
+import { useRef, useState, useEffect } from "react";
+import PostList from "./PostList";
+import Centered from "../UI/Centered";
+
+function FeedSection(props) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [loadedPosts, setLoadedPosts] = useState([]);
+  const [renderPosts, setRenderPosts] = useState(false); //used to rerender the sorted posts
+  const sorterRef = useRef();
+
+  function sortHandler() {
+    const chosenOption = sorterRef.current.value;
+    sortPosts(chosenOption);
+  }
+
+  function sortPosts(option) {
+    const compareAuthors = (itemA, itemB) => {
+      if (itemA.author.toLowerCase() < itemB.author.toLowerCase()) {
```

Comment:
I think this who sorting function could be better presented on the page to make it easier to read. Maybe think about combing sortHandler and sortPosts into one method. It would be cleaner not to nest the compareAuthors and Dates functions